Makefile

```diff
@@ -44,13 +44,31 @@ clean:
 clean-all: clean
 	rm -rf .vagrant
 
-check-righttoknow:
-	.venv/bin/ansible-playbook -i ./inventory/ec2-hosts site.yml -l righttoknow --check
+# Configure Keybase for MacOS
+macos-keybase:
+	ln -sf /Volumes/Keybase .keybase
```

Comment:
I don't *think* this should be needed.

I don't have a mac handy to test with, but my expectation was that the `keybase config get -d -b mountdir` command (line 6, this file) should work on a mac, and should return `/volumes/Keybase`. If that's right, this target shouldn't be needed - the same .keybase target should work on mac the same as it does on other *nix platforms.

I'm not objecting to this being added though - if it's needed, it's needed; if it's not strictly needed, it still adds a clearer alternative. I'm just surprised that the existing command didn't work.